### DISCUSSION TITLE
Support of oneOf, anyOf in OAS-3.0

### DISCUSSION THREAD

**Discussion by guptavip20** [OpenAPITools/openapi-diff#384] [Q&A] [answered]
Some of the specs, i am working with has keywords as oneOf, allOf and anyOf,  openapi-diff goes fine with allOf but doesn't work with anyOf, oneOf. Is it gonna be added in future? 

This is  the error  I am getting when testing with oneOf, anyOf:
  ```
 stack: 'SyntaxError: Swagger schema validation failed. \n' +
      "  Data does not match any schemas from 'oneOf' at #/paths//iam/v1/pets/{petId}/get/parameters/0\n" +
      "    Data does not match any schemas from 'oneOf' at #/paths//iam/v1/pets/{petId}/get/parameters/0\n" +
```

**Comment by joschi** [comment, marked as answer]
@guptavip20 Could you please provide the OpenAPI specification which is failing with the error message you've quoted?

The message also doesn't look like it was generated by this project. Are you sure you've been using _this_ `openapi-diff` and not another project of the same name?

**Reply by guptavip20** [reply]
Yes you are right, I have been using another tool but with the same name, I am using https://www.npmjs.com/package/openapi-diff. Apologies for inconvenience caused  :)

**Reply by joschi** [reply]
@guptavip20 No worries, we're always happy when people report bugs and help to investigate. 😄
